handel/src/aws/dynamodb-calls.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 Brigham Young University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import * as AWS from 'aws-sdk';
```

Comment:
We'll need to add some tests for the code in this module.

Reply:
I believe I added some test for the dynamo-calls. I'm not sure if it's sufficient. I have never used chai before, and was basically copying and pasting from other test classes. But I think I got the basics of the dynamo calls tested.